File: validate.js
const Ajv = require('ajv');
const schema = require('./person.schema.json');
const data = require('./person.json');

const ajv = new Ajv();
const validate = ajv.compile(schema);

const valid = validate(data);

if (valid) {
 console.log('JSON data is valid!');
} else {
 console.log('JSON data is invalid:', validate.errors);
}

File: src/dailyclimb/validate.py
import json

from jsonschema import validate


def validate_json(schema_file, json_file):
    with open(schema_file) as file:
        schema = json.load(file)

    with open(json_file) as file:
        data = json.load(file)

    validate(instance=data, schema=schema)

    print("JSON data is valid!")
